content/guides/additional/dist/dist.md

```diff
@@ -73,7 +73,7 @@ Suspending a desk turns off all its agents, saving their states in Gall. If ther
 
 Turning agents on and off is managed declaratively, rather than imperatively. Kiln maintains state for each desk about which agents should be forced on and which should be forced off. The set of running agents is now a function of the desk's `/desk/bill` manifest file and that user configuration state in Kiln. This means starting or stopping an agent is idempotent, both in Kiln and Gall.
 
-For details of the generators for managing desks and agents in Kiln, see the [`Dojo Tools`](/docs/userspace/dist/tools) document.
+For details of the generators for managing desks and agents in Kiln, see the [`Dojo Tools`](/using/os/dojo-tools) document.
```

Comment:
I was going to suggest moving or copying this doc over to duo as well.  What do you think?

Reply:
the using/os one, I mean

Reply:
the desk-related generator documentation isn't particular to developers. It was included in the software distribution docs cos  there were a bunch of changes to them that happened with the software distribution update so I just collected everything in the one place. I plan to flesh out the generator documentation to encompass all dojo utilities